src/runtime_src/xdp/profile/plugin/ocl/xocl_profile_cb.cpp

```diff
@@ -367,30 +367,6 @@ cb_action_ndrange_migrate (xocl::event* event,cl_int status, cl_mem mem0, size_t
     if (!isProfilingOn() || (totalSize == 0))
       return;
 
-    //Fix for CR-1004188 Using of the static variable below does not work
```

Comment:
hey Paul did you intend to remove this code

Reply:
Yes, we haven't used it in awhile.